### ISSUE TITLE
web client: do not express refresh interval in milliseconds

### ISSUE THREAD

**Issue by Morriz** [kiali/kiali#5234]
Nitpick, but consider people assuming seconds here: wouldn't the client hammer the api?

**Comment by jshaughn** [comment]
I agree that there isn't really a good reason for Duration to be in seconds and refreshInterval to be in millis; seconds would be more appropriate.  Having said that, I don't expect this will actually be an issue as the UI uses fixed values andI'd be surprised if anyone hand-manipulated the URL queryParam.  So, I'd accept a conttribution here, but it won't have priority for the maintainers.

**Comment by jshaughn** [comment]
Closing, no one has offered to pick this up and there are more important "help wanted" items.
